projects/angular/src/typography/STYLES.md

```diff
@@ -0,0 +1,10 @@
+# Lists
+
+## Class names
+
+| Class name    | Description                                             |
+| ------------- | ------------------------------------------------------- |
+| list          | Class that sets default sizes to list                   |
+| compact       | Class that sets list to use less space, used with .list |
+| list-unstyled | Class that removes default bullet styling of list items |
+| list-spacer   | Class that sets space before the list contents          |
```

Comment:
```suggestion
| list-spacer   | Class that sets space before the list contents          |
| list-group    |                                                         |
```
`list-group` seems to be missing.

Reply:
It looks like it is more of a use in the cards. Updated